ledger/participant-integration-api/src/main/scala/platform/store/Conversions.scala

```diff
@@ -183,4 +183,11 @@ private[platform] object Conversions {
     override val jdbcType: Int = ParameterMetaData.ByteArrayParameterMetaData.jdbcType
   }
 
+  // Array[String]
+
+  implicit object StringArrayParameterMetadata extends ParameterMetaData[Array[String]] {
```

Comment:
Is this one used by the current changes or it will be used in an upcoming one?

Reply:
Ah found it in `ContractWitnessesTable`, ignore this comment please.

Reply:
This was moved from its previous home in `RawEntry`, which has been removed.